src/Microsoft.AspNetCore.SignalR.Common/Internal/Formatters/BinaryMessageFormatter.cs

```diff
@@ -31,5 +31,44 @@ public static void WriteLengthPrefix(long length, IBufferWriter<byte> output)
 
             output.Write(lenBuffer.Slice(0, lenNumBytes));
         }
+
+        public static int WriteLengthPrefix(long length, Span<byte> output)
```

Comment:
Can the `IBufferWriter` variant above just use this now instead of duplicating code?

Reply:
Done